### PR TITLE
py-panaroo, py-edlib, and py-intbitset: new packages

### PR DESCRIPTION
**Pull request by luke-dt**

### CODE REVIEW

**Comment by tldahlgren** [comment]
There are some dependencies missing.  According to the unit tests, the following should be dependencies:  `py-edlib`, `py-intbitset`, and `py-sklearn`.

File: var/spack/repos/builtin/packages/py-sklearn/package.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PySklearn(PythonPackage):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-sklearn/package.py]: `sklearn` has been renamed to `scikit-learn`: https://pypi.org/project/sklearn/

You should use `scikit-learn` instead.

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-sklearn/package.py (hunk 1 above)
Comment (luke-dt) [reply var/spack/repos/builtin/packages/py-sklearn/package.py]: Yes, the developer of Panaroo probably should do that. Unfortunately for me, the depreciated sklearn is listed as a requirement in the panaroo setup.py.

It looks like it's only used for supplemental post-processing visualization (making a MDS plot):
https://github.com/gtonkinhill/panaroo/search?q=sklearn

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-sklearn/package.py]: I would still use `depends_on('py-scikit-learn')`. Even if the name is different, the imports are the same, so it should work properly with the new package dependency.

Comment (luke-dt) [reply var/spack/repos/builtin/packages/py-sklearn/package.py]: I didn't know if would just work, that's great! Re-built and confirmed that it functions on my local machine.

File: var/spack/repos/builtin/packages/py-intbitset/package.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    unsigned integers either 32-bits integers or an infinite range with fast
+    set operations implemented via bit vectors in a Python C extension for
+    speed and reduced memory usage."""
+
+    homepage = "https://github.com/inveniosoftware/intbitset"
+    pypi     = "intbitset/intbitset-3.0.1.tar.gz"
+
+    version('3.0.1', sha256='f1e6d03c6729922a223c51849df65b9e916e625aefb911784e7f9acd4c207d53')
+
+    depends_on('python@3.Z:',     type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-intbitset/package.py]: This line can be removed, they seem to support Python 2 and the Z doesn't mean anything here

File: var/spack/repos/builtin/packages/py-intbitset/package.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    depends_on('py-setuptools',   type='build')
+    depends_on('py-pytest',       type=('build'))
+    depends_on('py-pytest-xdist', type=('build'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-intbitset/package.py]: I don't see anything to suggest that pytest is required, these lines can be removed

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**